src/css-grids/grids.html

```diff
@@ -8,20 +8,32 @@
 </head>
 <body>
   <div class="grid-container">
-    <div class="box top-text">
+    <div class="box a">
+        <img src="fish.png"class="fishimg">
```

Comment:
get in the habit of writing `./` in front of the src link. 
`<img src="./fish.png" class="fishimg">`
do this for all your links